.github/workflows/inductor.yml

```diff
@@ -87,3 +87,35 @@ jobs:
       build-environment: linux-focal-py3_8-gcc7-build
       docker-image: ${{ needs.linux-focal-cpu-py3_8-gcc7-inductor-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-cpu-py3_8-gcc7-inductor-build.outputs.test-matrix }}
+
+  linux-bionic-cuda12_1-py3_10-gcc7-periodic-dynamo-benchmarks-build:
```

Comment:
As https://github.com/pytorch/pytorch/pull/98832 add CUDA 12.1 very recently.  Should we combine this with `linux-bionic-cuda11_8-py3_10-gcc7-inductor-build` with both running on CUDA 12.1?  Then both test matrixes could become one which looks nicer and also save a build.  We are moving toward CUDA 12.1 soon anyway I guess, so as long as the tests pass